src/constants/apiPaths.ts

```diff
@@ -1,9 +1,9 @@
 const API_PATHS = {
-  product: "https://.execute-api.eu-west-1.amazonaws.com/dev",
-  order: "https://.execute-api.eu-west-1.amazonaws.com/dev",
-  import: "https://.execute-api.eu-west-1.amazonaws.com/dev",
-  bff: "https://.execute-api.eu-west-1.amazonaws.com/dev",
-  cart: "https://.execute-api.eu-west-1.amazonaws.com/dev",
+  product: `https://jx5a3hc7j4..execute-api.eu-west-1.amazonaws.com/dev`,
```

Comment:
is this correct path with two dots? please check

Reply:
fixed, thanks!